server.js

```diff
@@ -72,15 +72,21 @@ bot.on('disconnected', () => {
 });
 
 function onMessage(msg) {
-  if (bot.user.username === msg.author.username) return;
-
   // Checks for PREFIX
   if (msg.content[0] === nconf.get('PREFIX')) {
-    let command = msg.content.toLowerCase().split(' ')[0].substring(1);
-    let suffix = msg.content.substring(command.length + 2);
-    let cmd = commands[command];
+    let cmd;
+    let command;
+    let suffix;
+    if (msg.content[1] === nconf.get('PREFIX')) {
+      command = 'append';
+      suffix = msg.content.substring(2);
+    } else {
+      command = msg.content.toLowerCase().split(' ')[0].substring(1);
+      suffix = msg.content.substring(command.length + 2);
+    }
+    cmd = commands[command];
 
-    if (cmd) callCmd(cmd, command, bot, msg, suffix);
+    if (cmd || msg.content[1] === nconf.get('PREFIX')) callCmd(cmd, command, bot, msg, suffix);
```

Comment:
You shouldn't need this line. You set `command` on line 81 so how it was before should be fine.